stable/konvoysoaktests/Chart.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+name: konvoysoaktests
+home: https://github.com/mesosphere/konvoy
```

Comment:
```suggestion
home: https://github.com/mesosphere/konvoy-soak
```
Maybe?

Reply:
no, the code will be living under konvoy to avoid having to manage go-k8s tests deps in konvoy-soak.

Reply:
Besides, it is the best place to validate any coming soak test